src/argilla/server/apis/v1/handlers/datasets.py

```diff
@@ -181,13 +181,13 @@ def get_current_user_dataset_metrics(
         "responses": {
             "count": datasets.count_responses_by_dataset_id_and_user_id(db, dataset_id, current_user.id),
             "submitted": datasets.count_responses_by_dataset_id_and_user_id(
-                db, dataset_id, current_user.id, ResponseStatusFilter.submitted
+                db, dataset_id, current_user.id, ResponseStatus(ResponseStatusFilter.submitted)
             ),
             "discarded": datasets.count_responses_by_dataset_id_and_user_id(
-                db, dataset_id, current_user.id, ResponseStatusFilter.discarded
+                db, dataset_id, current_user.id, ResponseStatus(ResponseStatusFilter.discarded)
             ),
             "draft": datasets.count_responses_by_dataset_id_and_user_id(
-                db, dataset_id, current_user.id, ResponseStatusFilter.draft
+                db, dataset_id, current_user.id, ResponseStatus(ResponseStatusFilter.draft)
```

Comment:
Un-related, but should we add this status back in the Python client too?

Reply:
Not yet for the draft one. We need to define how will be tackled from UI. Otherwise, users sending draft responses won't be able to see them from the UI